### PR TITLE
Improve HeartBeat events

### PR DESCRIPTION
**Pull request by Savolro**
* HeartBeat events are now sent once meshnet setting updates
* `is_on_vpn` value is always updated after connect and disconnect

### CODE REVIEW

File: events/moose/moose.go [hunk 1]
```diff
@@ -610,8 +622,11 @@ func (s *Subscriber) NotifyDisconnect(data events.DataDisconnect) error {
 			connectionTime, // seconds
 			"",
 			int32(-1),
-		))
+		)); err != nil {
+			return err
+		}
 	}
+	return s.response(moose.NordvpnappSetContextApplicationNordvpnappConfigCurrentStateIsOnVpnValue(false))
```
Comment (mariusSincovici) [inline events/moose/moose.go:629]: why is this needed if for connect was not?

**Review by mariusSincovici — APPROVED** [review]

File: events/moose/moose.go (hunk 1 above)
Comment (Savolro) [reply events/moose/moose.go:629]: I simply followed the logic that this has to be `true` only after regular VPN connection but after disconnect this value always has to be `false`. Yes, it would likely be correct if this was in the regular disconnect branch as it is never set to `true` from meshnet connections but I kept it here to cover more area for recover in case some misconfiguration happened.

**Review by bartoszWojciechO — APPROVED** [review]
